src/tests/particles/multi-slot-test.ts

```diff
@@ -1,92 +0,0 @@
-/**
- * @license
```

Comment:
should also delete the corresponding multi-slot-test.manifest file.

Reply:
Thank you, and thank you very much for the reviews in general.